tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
+        "checkJs": true,
+        "noEmit": true,
+        "target": "esnext",
+        "module": "commonjs",
+        "resolveJsonModule": true,
+        "strict": false
+    },
+    "include": [
+        "lib"
```

Comment:
```suggestion
        "es5"
```

Reply:
lib is the name of directory where the source lives. 

However, it's a good idea to explicitly exclude the dom so that the compiler can prevent use of those symbols. I'll do that.

Reply:
Actually, es6 is better than es5 I think.

Reply:
[`es6` includes `dom`](https://github.com/microsoft/TypeScript/blob/master/lib/lib.es6.d.ts#L22).

`es2015` is `es6` without `dom`.

Reply:
Huh. I thought es6 and es2015 were synonyms throughout TS. It certainly is for `target`. I opened #64 to fix that.